pkgs/data/fonts/fira-code/default.nix

```diff
@@ -2,17 +2,19 @@
 
 let
   version = "5.2";
+  packageRevision = "_1"; # TODO: remove this on next update
 in fetchzip {
-  name = "fira-code-${version}";
+  name = "fira-code-${version}${packageRevision}";
 
   url = "https://github.com/tonsky/FiraCode/releases/download/${version}/Fira_Code_v${version}.zip";
 
+  # only extract the variable font because everything else is a duplicate
   postFetch = ''
     mkdir -p $out/share/fonts
-    unzip -j $downloadedFile \*.ttf -d $out/share/fonts/truetype
+    unzip -j $downloadedFile '*-VF.ttf' -d $out/share/fonts/truetype
   '';
 
-  sha256 = "16v62wj872ba4w7qxn4l6zjgqh7lrpwh1xax1bp1x9dpz08mnq06";
+  sha256 = "1wbfjgvr9m5azl5w49y0hpqzgcraw6spd1wnxgxlzfx57x6gcw0k";
```

Comment:
Your statement

> it shares the same SHA256 hash.

appears to be false. I think this means you can drop the `packageRevision` part.

Reply:
My mistake. I probably forgot to update the hash on my initial try and the current package got reused because of it.

Reply:
I've now pushed a fix.